Redesigned About dialog

in3otd opened the pull request:
~~**Do not merge yet, this is still a WIP**~~
**Edit: cleaned up a bit the commits, should be ok now.**

Here is the redesigned About dialog, as per #172 . Please take a look at the dialog and code and check if it's ok. Might still need some (minor) fixes.

![qucsabout](https://cloud.githubusercontent.com/assets/9018179/7525597/0b81c3fa-f50b-11e4-844a-72359b286a35.png)

nvdl commented on qucs/qucs/dialogs/aboutdialog.cpp:
Possible this will be a resource reference later.

nvdl commented:
I was polishing the logo for the new dialog.
Possibly, you can add to it.

![logo](https://cloud.githubusercontent.com/assets/3097827/7526733/b2d2caf6-f514-11e4-8fe5-4bc9eb6c1412.png)

guitorri commented on qucs/qucs/dialogs/aboutdialog.h:
```diff
@@ -0,0 +1,61 @@
+  void setAuthorsText(void);
+  void setTrText(void);
+
+  std::array<QString, 8> currAuths; // current Qucs authors
```
`std::array` seems not yet available in clang 3.4... 
https://travis-ci.org/Qucs/qucs/jobs/61680030#L7991

in3otd replied:
oh, well... but with the clang 3.3 I'm using here it compiles fine (!). I think I forgot an `#include <array>` anyway, let me check...

guitorri commented:
We should unify the artwork on logos, icons and website (pick fonts and colors)... I scrapped some files from the mailing list I will put them somewhere accessible. However the ones I have are not SVG files... @nvdl which font did you use?

nvdl commented:
_polishing the logo_

It is already on the web, just added some gradient to give a 3D feel.
No idea about the font though. But if we agree on fonts, color schemes, etc. I can make new logos.

What is this stuff going on? Any Wiki (or something) where we can propose?

in3otd commented:
I have cleaned up the code a little, should be good to merge now, please check.

guitorri commented on qucs/qucs/dialogs/aboutdialog.cpp:
This link is dead.
I moved the component reference to https://github.com/guitorri/qucs-manual. Perhaps we can move it to the Qucs organization. It is still needs some work thou.
Perhaps the [Readme.md resources](https://github.com/Qucs/qucs#resources) would fit here nicely.

guitorri commented:
This looks great! 
Two other comments:
- @in3otd consider putting yourself on the list of authors;
- perhaps the font resize can be switched off on the QTextBrowser. I hit ctrl+scroll by accident and the text became very small...

in3otd commented:
ok, which link should I use then for the Component Reference Manual? Will you put it back on SF or keep it just in http://guitorri.github.io/qucs-manual/ ? It's a WIP, as many things here, but already quite good IMHO.
Will look into the latest two comments also.

guitorri commented:
I will move the qucs-manual to the Qucs organization. I just want to avoid publishing links that will soon be broken.

guitorri commented:
Done, moved to https://github.com/Qucs/qucs-manual

guitorri commented:
yes, the webpage: http://qucs.github.io/qucs-manual/

in3otd commented:
updated the authors list, updated the doc link and the text in the Support page and disabled the Ctrl-Wheel event. Took a while to find out who was actually handling it (not the QTextBrowser but its viewport). Should be ok now.

guitorri commented:
Very nice! Congrats!
